additional_tests/exchanges_tests/abstract_authenticated_exchange_tester.py

```diff
@@ -110,6 +110,8 @@ class AbstractAuthenticatedExchangeTester:
     IS_BROKER_ENABLED_ACCOUNT = True # set False when this test account can't generate broker fees
     # set True when this exchange used to have symbols that can't be traded through API (ex: MEXC)
     USED_TO_HAVE_UNTRADABLE_SYMBOL = False
+    # set true when IP whitleist test should be skipped even though EXCHANGE_IP_WHITELIST_ERRORS is set
+    SKIP_IP_WHITELIST_TEST = False
```

Comment:
😮

Reply:
this is when the local env doesnt have IP whitelisted API keys

Reply:
I just removed it to use a dynamic test skip